src/vms/pvm/etna-builder/builder.test.ts

```diff
@@ -950,6 +976,7 @@ describe('./src/vms/pvm/etna-builder/builder.test.ts', () => {
 
   describe('ImportTx', () => {
     it('should create an ImportTx with only AVAX and not non-AVAX assets', () => {
+      const gasPrice = 10_000n;
```

Comment:
I'm wondering if we should create a `FeeConfigResponse` fixture somewhere and re-use that in the tests